ecommerce/urls.py

```diff
@@ -3,5 +3,10 @@
 
 urlpatterns = [
     path('',views.Home.as_view(),name='home'),
+    path('products/', views.Products.as_view(),'products'),
```

Comment:
i think you forget to put the name attribute on third argument

Reply:
Solved!